### DISCUSSION TITLE
Why I can't see the motion blur in the reflected balls ?

### DISCUSSION THREAD

**Discussion by Skywalker13** [RayTracing/raytracing.github.io#969] [General]
Hello,

I don't understand why the motion blur is not "reflected" by the metal balls (looks not better with the glass balls). It's a bug or an optical effect that I don't understand?

![image](https://user-images.githubusercontent.com/397862/145964160-537e763c-c0db-4021-8afd-e75c9e8f5aa0.png)

Thank you

**Comment by hollasch** [comment]
Likely related to #799. This is fixed in `dev-major` (v4.0.0 release in development).

**Reply by Skywalker13** [reply]
Oh, Thank you..

**Reply by ssstroke** [reply]
I have implemented motion blur on January 11 2025 and the issue still persists.

UPD: I rendered the same scene in 1280x720 resolution and motion blur is present as it should be. Turns out it was not clearly visible on 400x225 resolution.
